src/js/_enqueues/lib/nav-menu.js

```diff
@@ -1063,12 +1086,35 @@
 		},
 
 		attachMenuSaveSubmitListeners : function() {
+			const updateNavMenuEl = $( '#update-nav-menu' );
+
 			/*
 			 * When a navigation menu is saved, store a JSON representation of all form data
 			 * in a single input to avoid PHP `max_input_vars` limitations. See #14134.
 			 */
-			$( '#update-nav-menu' ).on( 'submit', function() {
-				var navMenuData = $( '#update-nav-menu' ).serializeArray();
+			updateNavMenuEl.on( 'submit', function() {
```

Comment:
Remove all empty lines.

Reply:
- Resolved in https://github.com/WordPress/wordpress-develop/pull/6389/commits/24598ed85737201ee50112246c4c0b6da3458029 🚀 .